### PR TITLE
add cross compile tutorial

### PR DESCRIPTION
**Pull request by etrombly**
Added a tutorial for cross compiling from Linux to Windows

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Thanks so much about this!

@EPashkin: Seems ok for you? (I'll add the missing footer links once it's merged)

File: tuto/cross.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+The package installation steps are for arch, but should be similar for other distros.
```
Comment (EPashkin) [inline tuto/cross.md]: "Arch Linux" instead "arch" ?

File: tuto/cross.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Then set up the target in ~/.cargo/config
+
+    [target.x86_64-pc-windows-gnu]
+    linker = "x86_64-w64-mingw32-gcc"
+    ar = "x86_64-w64-mingw32-gcc-ar"
+
+Now install mingw and gtk libraries, this will take a while
+
+    pacaur -S mingw-w64-gcc mingw-w64-freetype2-bootstrap mingw-w64-cairo-bootstrap
```
Comment (EPashkin) [inline tuto/cross.md]: Maybe change it to `pacman`? Or default package manager changed?

**Comment by EPashkin** [comment]
@etrombly Thanks, now we will have documented answer to "How distribute gtk application?" 😉

File: tuto/cross.md (hunk 2 above)
Comment (etrombly) [inline tuto/cross.md]: mingw is in the AUR, so it won't work with pacman. I can try it in ubuntu in a bit so we can have instructions for both if you'd like.

Comment (EPashkin) [inline tuto/cross.md]: Understand now.
Can you add comment about this, if AUR don't installed by default?

**Comment by etrombly** [comment]
Updated the formatting a little, made the Arch section more clear. Also added instructions to add a program icon and change the theme to a windows 10 native look. If you want you should be able to adapt those sections pretty easily for someone building on windows, I haven't tried. Didn't end up adding Ubuntu instructions.

**Comment by GuillaumeGomez** [comment]
Thanks a lot! (you even added footer urls, awesome!)

**Pull request merged**